### PR TITLE
Added Project Euler Problems #1-3 and #5 in C

### PR DESCRIPTION
**Pull request by arnavb**
**Fixes issue:** N/A

**Changes:**
- Added Project Euler problems #1-3 and #5 in C

@abdouskamel @AdiChat

### CODE REVIEW

**Comment by arnavb** [comment]
@iattempt

File: code/online_challenges/src/project_euler/problem_001/problem_001.c [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#include <stdio.h>
+
+int
+main(void)
+{
+    int sum, i;
+    for (i = 0; i < 1000; ++i) {
```
Comment (YashovardhanSiramdas) [inline code/online_challenges/src/project_euler/problem_001/problem_001.c]: Braces for ```for``` loop are not required

**Comment by arnavb** [comment]
@YashovardhanSiramdas made the changes! @AdiChat Please merge!

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**